manta-pay/Cargo.toml

```diff
@@ -105,10 +105,10 @@ ark-ed-on-bls12-381 = { version = "0.3.0", optional = true, default-features = f
 ark-groth16 = { version = "0.3.0", optional = true, default-features = false }
 ark-snark = { version = "0.3.0", optional = true, default-features = false }
 ark-std = { version = "0.3.0", optional = true, default-features = false }
-bip32 = { version = "0.4.0", optional = true, default-features = false, features = ["bip39", "secp256k1"] }
+bip32 = { version = "0.3.0", optional = true, default-features = false, features = ["bip39", "secp256k1"] }
```

Comment:
Why use an older version?

Reply:
See the explanation in the PR description.